docs/gettingStarted/setupCustomSchemeAlchemy.md

```diff
@@ -1 +1,120 @@
-Coming Soon
+Schemes are the actions a DAOstack DAO can take. Once you have created your scheme contract and registered it to the DAO, you will have to add support for your scheme in various layers of the stack
```

Comment:
I think we might want to mention that there are two ways you can add an scheme + UI in alchemy:

- Add a new generic scheme and define an interface for that in alchemy, as documented here: https://github.com/daostack/alchemy/blob/f31d71207ef318dbbfe706b2eb5dc973b2f49e15/src/genericSchemeRegistry/README.md. 
- Create a completely new scheme from scratch, index it in the subgraph, and define an interface for it in alchemy

The first is much easier but less flexible.

Reply:
Moving this article under custom scheme tutorial and will add there